src/platform/site-wide/cta-widget/index.js

```diff
@@ -92,7 +91,7 @@ export class CallToActionWidget extends React.Component {
         ),
         buttonText: 'Sign In or Create an Account',
         buttonHandler: this.openLoginModal,
-        status: 'warning',
+        status: 'info',
```

Comment:
Doesn't really matter since this will be replaced by the appropriate category when using the updated `AlertBox` from Formation. Just following what the content had before this was implemented, which was info status.